### PR TITLE
Add load env feature

### PR DESCRIPTION
**Pull request by renanstn**
This PR adds a new feature: Given a path in Vault, it loads all values in environment variables on service start.

### CODE REVIEW

File: nameko_vault/__init__.py [hunk 1]
```diff
@@ -65,3 +68,40 @@ def delete_metadata_and_all_versions_kv_secret(
...
+    from Hashicorp Vault and inject those vars into the environment.
+    """
+
+    def __init__(self, mount_point: str, path: str):
+        self.mount_point = mount_point
+        self.path = path
+
+    def setup(self):
+        logging.info(
+            f"Importing env vars from Vault, mount_poit: '{self.mount_point}',"
```
Comment (arthurzeras) [inline nameko_vault/__init__.py]: Type error `mount_point`

File: nameko_vault/__init__.py [hunk 2]
```diff
@@ -65,3 +68,40 @@ def delete_metadata_and_all_versions_kv_secret(
...
+        env_vars = client.secrets.kv.read_secret_version(
+            mount_point=self.mount_point, path=self.path
+        )["data"]["data"]
+
+        logging.info("Vars loaded from Vault:")
+        for key, value in env_vars.items():
+            print(value)
+            logging.info(f"{key}: {value}")
+            os.environ[key] = value
+            print(os.getenv("VAR1"))
```
Comment (arthurzeras) [inline nameko_vault/__init__.py]: Why use print instead logging like before?

Comment (renanstn) [reply nameko_vault/__init__.py]: I forgot to delete these prints XD Sorry!

File: nameko_vault/__init__.py (hunk 1 above)
Comment (renanstn) [reply nameko_vault/__init__.py]: Thanks!

**Pull request merged**